.travis.yml

```diff
@@ -30,6 +27,9 @@ stages:
 
 jobs:
   include:
+    - name: "Linting with Go: 1.10"
+      script: ./.travis/lint.sh
+      go: "1.10"
```

Comment:
let's do the deploy and lint in the latest version (1.11), no?

Reply:
done